packages/form-js-viewer/package.json

```diff
@@ -8,6 +8,7 @@
       "require": "./dist/index.cjs"
     },
     "./dist/assets/form-js.css": "./dist/assets/form-js.css",
+    "./dist/assets/light.css": "./dist/assets/light.css",
```

Comment:
We definitely need to document this in the [viewer README](https://github.com/bpmn-io/form-js/tree/develop/packages/form-js-viewer) that it's required to import this extra style sheet from now on. Additionally, we'd need to
* [Adjust Playground docs](https://github.com/bpmn-io/form-js/tree/develop/packages/form-js-playground#styling)
* Create an issue to update the stylings in the [Camunda Form Playground](https://github.com/camunda/form-playground#styling)